src/pages/kiyoonkim/Login/Login.js

```diff
@@ -1,7 +1,62 @@
-import React from "react";
+import React, { useState } from "react";
+import "./Login.scss";
+import { useNavigate } from "react-router-dom";
 
-function Loginkimkiyoon() {
-  return <h1>this is login</h1>;
-}
+const Login = () => {
+  const navigate = useNavigate();
+  const [idValue, setIdValue] = useState("");
+  const [pwValue, setPwValue] = useState("");
 
-export default Loginkimkiyoon;
+  const saveUserId = function (event) {
+    setIdValue(idValue => {
+      idValue = event.target.value;
+      return idValue;
+    });
+  };
+
+  const saveUserPw = function (event) {
+    setPwValue(pwValue => {
+      pwValue = event.target.value;
+      return pwValue;
+    });
+  };
+
+  let loginIdPw = idValue.includes("@") && pwValue.length > 4;
+
+  return (
+    <div className="loginsection">
+      <p className="logo">Westagram</p>
+      <div className="inputsection">
+        <input
+          onChange={event => saveUserId(event)}
+          className="id-input"
+          type="text"
+          value={idValue}
+          placeholder="Phone number, username or email"
+        />
+        <input
+          onChange={event => saveUserPw(event)}
+          className="pw-input"
+          type="password"
+          value={pwValue}
+          placeholder="Password"
+        />
+        <button
+          onClick={() => {
+            navigate("/Mainkimyoon");
+          }}
+          className="loginbtn-act"
+          style={{ opacity: loginIdPw ? "1" : "0.3" }}
+          disabled={loginIdPw !== true}
```

Comment:
인라인 스타일 지양, 동적인 className 부여 혹은 :disabled 가상 선택자 활용
```suggestion
          disabled={!isloginIdPw}
```